src/kili/enums.py

```diff
@@ -175,7 +175,7 @@
 
 
 UploadType = Literal[
-    "VIDEO",
+    "VIDEO_LEGACY",
```

Comment:
This should stay as "VIDEO". It is an enum that has two type for asynchronous upload: VIDEO or GEO_SATELLITE